app.js

```diff
@@ -2,6 +2,11 @@ const Emitter = require('events');
 const assert = require('assert');
 const transform = require('sdp-transform');
 const { v4: uuidv4 } = require('uuid');
+var BoundaryTag="--uniqueBoundary"
```

Comment:
please end all statements with semi-colon.  Also, do not use var to declare a variable

Reply:
before committing a change, please run `npm run jslint` and verify there are no errors